e2e/callback_test.go

```diff
@@ -195,351 +196,322 @@ func (suite *CbTestSuite) SetupTest() {
-		GetSentCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerSentQuery, &resp)
-	require.NoError(t, err)
-	return *resp.Owner
-}
-
-func queryTesterReceived(t *testing.T, chain *wasmibctesting.TestChain, tester string) string {
-	resp := TesterResponse{}
-	testerReceivedQuery := TesterReceivedQuery{
-		GetReceivedCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	require.NoError(t, err)
-	return *resp.Owner
-}
-
-func queryTesterNftContract(t *testing.T, chain *wasmibctesting.TestChain, tester string) string {
-	resp := ""
-	testerReceivedQuery := TesterNftContractQuery{
-		GetNftContract: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	require.NoError(t, err)
-	return resp
-}
-
-func queryTesterReceivedErr(t *testing.T, chain *wasmibctesting.TestChain, tester string) error {
-	resp := TesterResponse{}
-	testerReceivedQuery := TesterReceivedQuery{
-		GetReceivedCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	return err
-}
-
-func (suite *CbTestSuite) TestSuccessfulTransfer() {
+func (suite *CallbackTestSuite) TestSuccessfulTransfer() {
 	memo := callbackMemo(nftSentCb(), "", nftReceivedCb(), "")
+	// A -> B token_id 2
 	sendIcsFromChainAToB(suite, suite.cw721A.String(), "2", memo, true)
 
 	// Query the owner of NFT on cw721
-	chainAOwner := queryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "2")
+	chainAOwner := test_suite.QueryGetOwnerOf(suite.T(), suite.chainA, suite.cw721A.String(), "2")
 	require.Equal(suite.T(), chainAOwner, suite.bridgeA.String())
-	chainBOwner := queryGetOwnerOf(suite.T(), suite.chainB, suite.cw721B.String(), "2")
+	chainBOwner := test_suite.QueryGetOwnerOf(suite.T(), suite.chainB, suite.cw721B.String(), "2")
 	require.Equal(suite.T(), chainBOwner, suite.testerB.String())
 
 	// We query the data we have on the tester contract
 	// This ensures that the callbacks are called after all the messages was completed
 	// and the transfer was successful
-	testerDataOwnerA := queryTesterSent(suite.T(), suite.chainA, suite.testerA.String())
+	testerDataOwnerA := test_suite.QueryTesterSent(suite.T(), suite.chainA, suite.testerA.String())
 	require.Equal(suite.T(), testerDataOwnerA, suite.bridgeA.String())
-	testerNftContract := queryTesterNftContract(suite.T(), suite.chainB, suite.testerB.String())
+	testerNftContract := test_suite.QueryTesterNftContract(suite.T(), suite.chainB, suite.testerB.String())
 	require.Equal(suite.T(), testerNftContract, suite.cw721B.String())
-	testerDataOwnerB := queryTesterReceived(suite.T(), suite.chainB, suite.testerB.String())
+	testerDataOwnerB := test_suite.QueryTesterReceived(suite.T(), suite.chainB, suite.testerB.String())
 	require.Equal(suite.T(), testerDataOwnerB, suite.testerB.String())
 }
 
-func (suite *CbTestSuite) TestSuccessfulTransferWithReceivers() {
+func (suite *CallbackTestSuite) TestSuccessfulTransferWithReceivers() {
```

Comment:
This test has same result as for `TestSuccessfulTransfer`. By its name, I wouldnt know what it actually tests. As far as I see only difference is, by memo, where addresses are provided in one and not provided by other test:

TestSuccessfulTransferWithReceivers:
memo := callbackMemo(nftSentCb(), suite.testerA.String(), nftReceivedCb(), suite.testerB.String())

TestSuccessfulTransfer:
memo := callbackMemo(nftSentCb(), "", nftReceivedCb(), "")